test/Extensions/ServiceBus.Tests/StatisticMonitorTests/EHStatisticMonitorTests.cs

```diff
@@ -113,25 +113,29 @@ public EHStatisticMonitorTests(Fixture fixture)
 
         private void AssertCacheMonitorCallCounters(CacheMonitorCounters totalCacheMonitorCallCounters)
         {
-            Assert.True(totalCacheMonitorCallCounters.TrackCachePressureMonitorStatusChangeCallCounter > 0);
-            Assert.True(totalCacheMonitorCallCounters.TrackMemoryAllocatedCallCounter > 0);
-            Assert.Equal(0, totalCacheMonitorCallCounters.TrackMemoryReleasedCallCounter);
-            Assert.True(totalCacheMonitorCallCounters.TrackMessageAddedCounter > 0);
-            Assert.Equal(0, totalCacheMonitorCallCounters.TrackMessagePurgedCounter);
+            var c = totalCacheMonitorCallCounters;
+            Assert.True(c.TrackCachePressureMonitorStatusChangeCallCounter > 0,
+                $"Expected {nameof(c.TrackCachePressureMonitorStatusChangeCallCounter)} > 0, got {c.TrackCachePressureMonitorStatusChangeCallCounter}");
+            Assert.True(c.TrackMemoryAllocatedCallCounter > 0, $"Expected {nameof(c.TrackMemoryAllocatedCallCounter)} > 0, got {c.TrackMemoryAllocatedCallCounter}");
+            Assert.True(0 == c.TrackMemoryReleasedCallCounter, $"Expected {nameof(c.TrackMemoryReleasedCallCounter)} == 0, got {c.TrackMemoryReleasedCallCounter}");
+            Assert.True(c.TrackMessageAddedCounter > 0, $"Expected {nameof(c.TrackMessageAddedCounter)} > 0, got {c.TrackMessageAddedCounter}");
+            Assert.True(0 == c.TrackMessagePurgedCounter, $"Expected {nameof(c.TrackMessagePurgedCounter)} == 0, got {c.TrackMessagePurgedCounter}");
```

Comment:
During debugging, I found that there was not sufficient information in these errors (and lines didn't seem to match correctly), so I add more here, but these changes are not related to the core of this PR